src/textinput/TextInput.ts

```diff
@@ -149,23 +117,11 @@ export default class TextInput extends TextInputBase<TextInputProperties> {
 				ontouchend: this._onTouchEnd,
 				ontouchcancel: this._onTouchCancel
 			})
```

Comment:
I think we should actually keep `inputWrapper`. It's pretty necessary for anything but basic CSS styles since you can't add pseudo elements to an `<input>`. TextInput addons would need it, for example.